docs/src/getting-started/basics.md

```diff
@@ -14,30 +25,30 @@ let server = FuelService::new_node(Config::local_node()).await.unwrap();
 let client = FuelClient::from(srv.bound_address);
 ```
 
-Alternatively, if you have a Fuel node running separately, you can pass in the `SocketAddr`
-to `FuelClient::from()`.
-
-It's important to setup this client, as it will be needed later when instantiating contracts. More
-on that on the section below.
+Alternatively, if you have a Fuel node running separately, you can pass in the `SocketAddr` to `FuelClient::from()`.
 
-### Deploying a Sway contract
+## Deploying a Sway contract
 
-Once you have a Fuel node running and the compiled contract in hands, it's time to deploy the
-contract:
+Once you have compiled your contract, here's how to deploy it:
 
 ```Rust
-let salt: [u8; 32] = rng.gen();
-let salt = Salt::from(salt);
-// Setup a local node
-let server = FuelService::new_node(Config::local_node()).await.unwrap();
+use fuels::prelude::*;
+use fuels_abigen_macro::abigen;
+
+// This will generate your contract's methods onto `MyContract`.
+// This means an instance of `MyContract` will have access to all
+// your contract's methods that are running on-chain!
+abigen!(
+    MyContract,
+    "your_project/out/debug/contract_test-abi.json",
+);
+
+// This helper will setup a local node/provider and a test wallet.
 let (provider, wallet) = setup_test_provider_and_wallet().await;
 
 // Load the compiled Sway contract (this is the output from `forc build`)
-let compiled = Contract::load_sway_contract(
-"your_project/out/debug/contract_test.bin",
-salt,
-)
-.unwrap();
+let compiled = MyContract::load_sway_contract(
```

Comment:
With my incoming PR for #217  this will be handled by the binding but for now it works via `Contract`

```suggestion
let compiled = Contract::load_sway_contract(
```

Reply:
Yup, that was a typo!